### PR TITLE
feat: added landing page

### PR DESCRIPTION
**Pull request by TejasShekar**
Added a simple landing page

https://tejasshekar.github.io/P8-Gamez/

### CODE REVIEW

File: home.css [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+}
+
+.btn-primary a {
+  color: var(--secondary);
+  font-size: x-large;
+}
+
+.overlay {
+  position: absolute;
+  background: rgba(0, 0, 0, 0.6);
```
Comment (ravikumar0403) [inline home.css:45]: use variable here as well

**Review by ravikumar0403 — APPROVED** [review]
Added one comment, rest all looks good

**Review by HarshnaNHaswani — APPROVED** [review]
Good to go, Just a suggestion don't  mix buttons and icons in header nav

**Review by sanjay-xdr — APPROVED** [review]
Code looks good just a suggestion that try to create a class for properties which you are using again and again like flex property
everything else looks good

**Comment by TejasShekar** [comment]
> Code looks good just a suggestion that try to create a class for properties which you are using again and again like flex property everything else looks good

Note`

**Pull request merged**